docs/src/userguide/faq.rst

```diff
@@ -0,0 +1,1018 @@
+
+----------
+
+What Python versions does Cython support?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Answer**: From Cython 0.21 on, the supported versions are 2.6, 2.7 and 3.4+. Cython 3.0 removes support for Python 2.6 and requires either Python 2.7 or Python 3.4+. Python 2.x support is scheduled for removal in Cython 3.1, which will probably require Python 3.6 or later at the time of its release.
+
+The C code generated by Cython is portable and builds in all supported Python versions. All supported CPython release series are tested regularly. New CPython versions are usually supported before they are released.
+
+The source code that Cython compiles can use both Python 2 and Python 3 syntax, defaulting to Python 2 syntax in Cython 0.x and Python 3 syntax in Cython 3.x and later. When compiling Cython modules (.pyx files) in Python 2 mode, most Python 3 syntax features are available by default if they do not interfere with Python 2 syntax (as in Python 2.7), but the general language semantics are defined as in Python 2. When compiling Python modules (.py files), the special Cython syntax (such as the ``cdef`` keyword) is not available. For both input types, the language level can be set to Python 3 by either passing the "-3" option to the compiler, or by putting
+
+::
+
+    # cython: language_level=3
+
+at the top of the module file (within the first comment and before any code or empty lines). With Cython 3.x, compiling Python 2 code requires the option "-2" or the directive ``language_level=2``. The most visible difference is that unprefixed strings are unicode strings under Python 3 semantics, but this also makes ``print()`` a function, keeps loop variables in list comprehensions from leaking into the outer scope, etc. To get Python 3 syntax features without enforcing unicode literals, set ``language_level=3str`` or use the option ``--3str``.
+
+----------
+
+Can I place the output under the BSD license, or does it have to be the python-license as well?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Answer**: You can use the output of Pyrex/Cython however you like (and license it how you like - be it BSD, public domain, GPL, all rights reserved, whatever).
+
+More details: The Python License is different from the GPL used for GCC, for example. GCC requires a special exception clause for its output as it is *linked* against the library part of GCC, i.e. against GPL software, which triggers the GPL restrictions.
+
+Pyrex doesn't do anything similar, and linking against Python is not restricted by the Python License, so the output belongs to the User, no other rights or restrictions involved.
```

Comment:
```suggestion
Pyrex/Cython doesn't do anything similar, and linking against Python is not restricted by the Python License, so the output belongs to the User, no other rights or restrictions involved.
```

Reply:
I think most of the pyrex stuff can be removed, TBH (perhaps except in the "About the project" section for historical interest).